rdm/md_extensions/rdm_extension.py

```diff
@@ -0,0 +1,45 @@
+from jinja2 import nodes
+from jinja2.ext import Extension
+
+from rdm.util import post_processing_filter_list
+
+
+class RdmExtension(Extension):
+
+    def __init__(self, environment):
+        super().__init__(environment)
+
+        # add the post processing list to the environment.
+        if not hasattr(environment, 'rdm_post_process_filters'):
```

Comment:
It seems like it would be clearer to add:

```
environment.extend(rdm_post_process_filters=[])
```

in `render.py`, instead of depending on the first extension class to add it.

See my other comment.